providers/yttcb/clusterbootstrap.yaml

```diff
@@ -16,6 +19,33 @@
 #@   end
 #@ end
 
+#@ def pinniped_configs_exist():
+#@   return (data.values.IDENTITY_MANAGEMENT_TYPE != "none" and
```

Comment:
There are 2 other CERT duration types of variables that Pinniped understands, but alone do not contribute meaningfully to Pinniped package, so I left them out of this function.

Reply:
Josh had a goo question... what is the default value of data.values.IDENTITY_MANAGEMENT_TYPE in this yttcb context? Is it "none" or "" or something else? There is no values.yaml file in the providers/yttcb directory.

Reply:
This is indeed a great question!

Reply:
We ended up writing the code to specifically check for "ldap" or "oidc". Any other value will disable Pinniped.

We also decided to only check for the data.values.IDENTITY_MANAGEMENT_TYPE value. If it is either of the supported values, then assume that the user is trying to configure Pinniped.